.goreleaser.yml

```diff
@@ -0,0 +1,85 @@
+---
+version: 2
+project_name: enduro-sdps
+
+changelog:
+  sort: asc
+  filters:
+    exclude:
+    - '^docs:'
+    - '^website:'
```

Comment:
Do we really need the changelog here? Is this going to the Github release page? If so, I think we should disable it for now. Otherwise, I don't like it sorting the commits by message and I we don't really use these patterns in the commit messages.

Reply:
done